ccdb/settings_production.py

```diff
@@ -1,6 +1,9 @@
-# flake8: noqa
```

Comment:
@nikolas I was expecting to see errors after I pulled this, but I didn't. It builds cleanly on my machine, though. Should I leave this alone?

Reply:
Cool, sounds good. I think the reason we had the noqa in here is just to ignore the syntax error caused by the templating here: https://github.com/ccnmtl/ctldjango/blob/master/project_name/settings_production.py#L2

So the plan is to remove the global noqa, and only add in specific ignores here flake8 needs them, like this: https://github.com/ccnmtl/ctldjango/pull/325/files

Reply:
Looks like travis caught some errors here though, so we will have to add in the specific ignores. I'm seeing them too - use `make flake8` to just run the flake8 check.